css/_global.scss

```diff
@@ -40,3 +42,15 @@ button,a {
     text-decoration: none;
   }
 }
+a{
+  @media screen and (min-width:701px) {
+    font-size: 26px;
+    text-decoration: none;
+    font-family: 'Sawarabi Gothic';
+  }
+  @media screen and (max-width:700px) {
+    font-size: 16px;
+    text-decoration: none;
+    font-family: 'Sawarabi Gothic';
+  }
```

Comment:
@PinhoOn if they have same font-family and text-decoration you can move both outside the `@media` to increase performance